mods/cnc/rules/tech.yaml

```diff
@@ -19,7 +38,7 @@ V19.Husk:
 		Footprint: x
 		Dimensions: 1,1
 	Tooltip:
-		Name: Oil Derrick (Destroyed)
+		Name: Oil Derrick (Recapturable)
```

Comment:
In any case, please do not change the tooltips.  They are descriptive, and it *is* a destroyed oil derrick.